Sources/Authentication/AppAuthSession.swift

```diff
@@ -75,7 +75,7 @@ public final class AppAuthSession: LoginSession {
         flow?.resumeExternalUserAgentFlow(with: url)
         
         guard let authorizationCode else {
-            // todo: are there specific values of `self.error` we care about here?
+            // are there specific values of `self.error` we care about here?
```

Comment:
I assume the answer is no, in which case we should remove the comment.. 
or yes in which case we should raise a ticket and keep the `todo`? @amopatel1234